### PR TITLE
Only show connected addrs for peers in swarm.peers

### PR DESCRIPTION
**Pull request by VictorBjelkholm**

### CODE REVIEW

File: src/core/components/swarm.js [hunk 1]
```diff
@@ -24,24 +23,19 @@ module.exports = function swarm (self) {
...
-
-      const peerList = flatMap(peers, (peer) => {
-        return peer.multiaddrs.toArray().map((addr) => {
+        .map((peer) => {
+          const connectedAddr = peer.isConnected()
           const res = {
-            addr: addr,
-            peer: peer
+            addr: connectedAddr,
+            peer
```
Comment (diasdavid) [inline src/core/components/swarm.js]: please keep using the notation `peer: peer`.

**Comment by diasdavid** [comment]
@VictorBjelkholm what's missing to get his merged?

**Comment by VictorBjelkholm** [comment]
@diasdavid needs https://github.com/ipfs/interface-ipfs-core/pull/150 to be merged + released to have the tests, otherwise this PR is ready. Just pushed update removing the object literal shorthand.

**Comment by diasdavid** [comment]
@VictorBjelkholm seems that it is failing:

```
          Shows connected peers only once
Swarm listening on /ip4/127.0.0.1/tcp/37115/ipfs/QmfC6YXRVc6PHVtWVTWnbrFKzNDrYPBHEweq8MYrEsoN1q
              1) Connecting two peers with one address each
Swarm listening on /ip4/127.0.0.1/tcp/6543/ipfs/QmdfdpQjfrp6gi3wJCBQ2TpNmxgnkwTmQ2w7BsqwjD8vin
Swarm listening on /ip4/127.0.0.1/tcp/6544/ipfs/QmdfdpQjfrp6gi3wJCBQ2TpNmxgnkwTmQ2w7BsqwjD8vin
              2) Connecting two peers with two addresses each
```

**Comment by diasdavid** [comment]
rebased master onto this branch

**Comment by diasdavid** [comment]
@VictorBjelkholm tests pass locally but not in CI

image

**Pull request merged**